modules/virt-temporary-token-VNC.adoc

```diff
@@ -27,14 +27,15 @@ $ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} --type json
 # ...
 ----
 
-. Generate a token by running the following command:
+. Generate a token:
```

Comment:
We now use _by running_ phrasing. See the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command).

```suggestion
. Generate a token by running the following command:
```